docs/generated/sql/bnf/stmt_block.bnf

```diff
@@ -3044,10 +3050,10 @@ family_name ::=
 	name
 
 reference_on_update ::=
-	'ON' 'UPDATE' reference_action
+	'ON_UPDATE' 'UPDATE' reference_action
```

Comment:
sigh, i rarely read generated files on reviews but maybe i should
sorry to spring this on you also, but do you know why this isn't ON_UPDATE_LA?

Reply:
hmm, i guess the generated code omits the `_LA` as the other ones don't seem to be mentioned here.
maybe to make this clearer, it should be `ON_FK_UPDATE`

Reply:
thinking about this more, this might actually have to be just `ON_LA` so that the bnfs we use in our railroad diagrams in our docs just show `ON`. or we change the generated code to make this nicer...

Reply:
Oh good call. I just have to disambiguate between FK and ON UPDATE, so ON_LA for the FK use cases works here.

Reply:
you still need a `make buildshort` to update this